src/views/splash/splash.jsx

```diff
@@ -215,11 +215,11 @@ var Splash = injectIntl(React.createClass({
                 <Box
                         key="curator_top_projects"
                         title={
-                            'Projects Curated by ' +
+                            formatMessage({id: 'splash.projectsCuratedBy', defaultMessage: 'Projects Curated by '}) +
```

Comment:
Good to know. I was just using the following 'Learn More' message as a template.